libraries/chain/database.cpp

```diff
@@ -2557,9 +2557,9 @@ namespace golos { namespace chain {
        /**
         *  At a start overall the network has an inflation rate of 15.15% of virtual golos per year.
         *  Each year the inflation rate is reduced by 0.42% and stops at 0.95% of virtual golos per year in 33 years.
-        *  66.67% of inflation is directed to content reward pool
-        *  26.67% of inflation is directed to vesting fund
-        *  6.66% of inflation is directed to witness pay
+        *  60.003% of inflation is directed to content reward pool
+        *  24.003% of inflation is directed to vesting fund
+        *  5.994% of inflation is directed to witness pay
```

Comment:
actually, we have only 2 fractional digits in percent type. and sum gives 90%, please, describe missing 10% too